src/odbc/unittests/connect.c

```diff
@@ -12,18 +12,25 @@ init_connect(void)
 
 #ifdef _WIN32
 #include <odbcinst.h>
+#undef SQLGetPrivateProfileString
+#if !HAVE_SQLGETPRIVATEPROFILESTRING
+#  define SQLGetPrivateProfileString tds_SQLGetPrivateProfileString
+int tds_SQLGetPrivateProfileString(LPCSTR pszSection, LPCSTR pszEntry,
+                                   LPCSTR pszDefault, LPSTR pRetBuffer,
+                                   int nRetBuffer, LPCSTR pszFileName);
```

Comment:
What happens if we use a DLL here? I would assume this test will refer to an undefined symbol and fail to load.

Reply:
Ah, yeah, (conditional) exporting is in order here.  However, I need a static build anyway for the sake of the `all_types`, `connection_string_parse`, and `utf8_4` tests, so our Windows builds leave this library static-only for simplicity.

Reply:
Tried to sort out this last hunk. Maybe code for that specific function could be just duplicated somehow? I tried to build for Windows and `SQLGetPrivateProfileString` from odbc system libraries was used, not the version from shared library.